test/test_daemon_start.py

```diff
@@ -195,6 +198,26 @@ def test_config_and_start_and_stop(self):
         self.stop_daemon(d)
         self.assertFalse(os.path.exists(d.pidfile))
 
+        # Start as a daemon
```

Comment:
You are a doing two time the same thing actually. The following line are copy paste from the previous ones.